src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -316,6 +316,91 @@ struct P1_ALIGNAS(4) RawIMUOutput : public MessagePayload {
   int32_t gyro[3] = {INT32_MAX, INT32_MAX, INT32_MAX};
 };
 
+/**
+ * @brief IMU sensor measurement input  with calibration and corrections applied
+ *        (@ref MessageType::IMU_INPUT, version 1.0).
+ * @ingroup measurement_messages
+ *
+ * This message is an input  from the device containing IMU acceleration and
+ * rotation rate measurements. The measurements been corrected for biases and
+ * scale factors, and have been rotated into the vehicle body frame from the
+ * original IMU orientation, including calibrated mounting error estimates.
+ *
+ * See also @ref RawIMUInput.
+ */
+struct P1_ALIGNAS(4) IMUInput : public MessagePayload {
```

Comment:
See below - let's remove this class in favor of the raw format below, and rename that one.

Reply:
Alright I think that makes sense, I was thinking this was going to make users confused lol.